scalafix-core/shared/src/main/scala/scalafix/internal/rule/NoInfer.scala

```diff
@@ -1,28 +1,41 @@
 package scalafix.internal.rule
 
 import scala.meta._
+import metaconfig.{Conf, Configured}
 import scalafix.lint.LintCategory
 import scalafix.lint.LintMessage
-import scalafix.rule.RuleCtx
+import scalafix.rule.{Rule, RuleCtx}
 import scalafix.rule.SemanticRule
 import scalafix.util.SemanticdbIndex
 import scalafix.util.SymbolMatcher
+import scalafix.internal.config.Config
 
-case class NoInfer(index: SemanticdbIndex)
+final case class NoInfer(index: SemanticdbIndex, configuration: Config[NoInfer])
     extends SemanticRule(index, "NoInfer")
     with Product {
-  private val badSymbol = SymbolMatcher.exact(NoInfer.badSymbols: _*)
-  val error: LintCategory =
+
+  private lazy val error: LintCategory =
     LintCategory.error(
       """The Scala compiler sometimes infers a too generic type such as Any.
         |If this is intended behavior, then the type should be explicitly type
         |annotated in the source.""".stripMargin
     )
+
+  private lazy val noInferSymbol: SymbolMatcher =
+    if (configuration.symbols.isEmpty)
+      SymbolMatcher.exact(NoInfer.badSymbols: _*)
```

Comment:
`.normalized` instead of `.exact` will allow you to write `scala.Predef.any2stringadd` in the config ;)

Reply:
I will try normalized, then. Like I did with Disable. Many thanks